components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.ts

```diff
@@ -66,6 +73,32 @@ export class DesktopDetailComponent implements OnInit, OnDestroy {
     this.getCheckInHistory();
   }
 
+  onDownloadCheckInHistory(format) {
+    this.closeDownloadDropdown();
+    const filename = `${moment.utc().format(DateTime.REPORT_DATE_TIME)}.${format}`;
+
+    const onComplete = () => console.warn('completed downloading report');
+    const onError = _e => console.error('error downloading report');
```

Comment:
Seems like these would be better served as notification ribbons that say success/failure?